app/controllers/organisation_relationships_controller.rb

```diff
@@ -18,12 +18,36 @@ def housing_providers
     end
   end
 
+  def add_housing_provider
+    organisations = Organisation.all
+    respond_to do |format|
+      format.html do
+        @organisations = organisations
+        render "organisation_relationships/add_housing_provider", layout: "application"
+      end
+    end
+  end
+
+  def create
```

Comment:
i think we can have a #create_housing_provider and a #create_managing_agent, they could directly create the org relationship or could call #create with the relevant params

Reply:
this is needed because the child/parent relationship will be swapped and the type changed